sema/check/check.py

```
@@ -0,0 +1,61 @@
+        """Initialize the Check Service object
+
+        :param input_folder: the folder where the files to be checked are located
+        :type input_folder: str
+        :param output: the output format to be used
+        :type output: str
+        """
+        self.input_folder = input_folder
+        self.output = output
+
+        log.debug(
+            f"Check service initialized with input_folder: {input_folder}, output: {output}"
```

Ruff

Logging statement uses f-string

(G004)